frontend/doc-site/04__Analyze Public Data/4_4__Annotate.mdx

```diff
@@ -48,47 +45,63 @@ conda activate cellxgene
 ```
 
 ### pip install
-Once your virtual environment is installed and activated, install the `annotate` feature using pip using the following command:
+
+Once your virtual environment is installed and activated, install the `annotate` feature using pip with the following command:
 
 `pip install 'cellxgene[annotate]'`
 
+If `conda` is not installed on your system, install these additional packages:
+`pip install scvi-tools==0.16.2 'scanpy[leiden]==1.9.1' xgboost==1.6.1 torch==1.11.0`
+If this fails, we recommend you use a Conda environment (see the [conda setup](#conda-setup) section, above)
+
 You can run the following to test the success of your installation:
 
-`cellxgene annotate -h` or `cellxgene annotate --help`
+`cellxgene annotate --help`
 
 ## CELLxGENE Annotation Workflow
 
-See our quickstart guide to  quickly see if you can run through the pipeline or check out the more detailed sections below for more detailed descriptions of the workflow. Both assume that you have properly set up a clean virtual environment to work from.
+See our quickstart guide to quickly see if you can run through the pipeline or check out the more detailed sections below for more detailed descriptions of the workflow. Both assume that you have properly set up a clean virtual environment to work from.
 
 ### Quickstart
 
 In this quickstart, we will use our lung model (trained on the [Integrated Human Lung Cell atlas](https://cellxgene.cziscience.com/collections/6f6d381a-7701-4781-935c-db10d30de293)), to make annotations on a subset of another human lung dataset, [LungMap](https://cellxgene.cziscience.com/collections/625f6bf4-2f33-4942-962e-35243d284837). You can download the entire LungMap dataset [here](https://cellxgene.cziscience.com/collections/625f6bf4-2f33-4942-962e-35243d284837) Depending on your environment manager, execute one of the following quickstart commands:
 
 #### venv
+
 ```
-mkdir annotations  # create a directory to download data, run annotation pipeline and store annotated object
+# create a directory to download data, run annotation pipeline and store annotated object
+mkdir annotations
```

Comment:
"comments above command" seems cleaner; feel free to disagree--it does make it vertically longer